package.json

```diff
@@ -18,7 +18,10 @@
     "ava": "ava --verbose --extension ts",
     "ts-node": "ts-node",
     "dist": "npm run clean && tsc && jq \"del (.files)\" < package.json > dist/package.json && shx cp src/puppet-web/*.js dist/src/puppet-web/",
-    "doc": "npm run dist && echo '# Wechaty v'$(jq -r .version package.json)' Documentation\n' > docs/index.md && jsdoc2md dist/src/{wechaty,room,contact,friend-request,message}.js dist/src/puppet-web/friend-request.js>> docs/index.md",
+    "doc": "npm run dist && echo '# Wechaty v'$(jq -r .version package.json)' Documentation\n' > docs/index.md && npm run doc-api && npm run doc-readme",
+    "doc-readme": "npm run doc-index && ts-node docs/update-readme.ts && shx rm -rf docs/doc-index.md",
+    "doc-index": "jsdoc2md --partial docs/partials/*.hbs --template docs/api-index.hbs dist/src/{wechaty,room,contact,friend-request,message}.js dist/src/puppet-web/friend-request.js>> docs/doc-index.md",
```

Comment:
Please:
1. rename `doc-index`  to `doc:toc`
1. output `doc:toc` to `stdout` instead of writing it to file, because its output is for temporary usage.